pyaerocom/aeroval/setupclasses.py

```diff
@@ -246,6 +246,7 @@ def __init__(self, exp_id: str, **kwargs):
         self.exp_descr = ""
         self.public = False
         self.exp_pi = getuser()
+        self.pyaerocom_version = __version__
```

Comment:
Why is `self.pyaerocom_version` needed?
Will it end up somewhere on the JSON?

Reply:
Yes, the idea is that by including it in the JSON, @AugustinMortier determine from version how to read in the json files in hm/ts (i.e., the old convention with statistics_ts.json or the proposed method)